<!-- Launch File for running svo with ceres backend from bag-file -->
<launch>
  
  <arg name="type" default="vio"/>
  
  <arg name="calib_file" default="$(find svo_ros)/param/calib/realsense/realsense_rgbd.yaml"/>

  <arg name="config_file" default="$(find svo_ros)/param/realsense_vio_rgbd.yaml"/>

  <!-- SVO node -->
  <node pkg="svo_ros" type="svo_node" name="svo" clear_params="true" output="screen" args="--v=0">

    <!-- Camera topic to subscribe to -->
    <param name="image_topic" value="/camera/color/image_raw" type="str" />
    <param name="depth_topic" value="/camera/aligned_depth_to_color/image_raw" type="str" />
	
    <!-- Imu topic to subscribe to -->
    <param name="imu_topic" value="/camera/imu" type="str" />

    <!-- Odom topic to subscribe to -->
    <param name="odom_topic" value="/odom" type="str" />

    <!-- Camera calibration file -->
    <param name="calib_file" value="$(arg calib_file)" />

    <!-- Config file -->
    <param name="config_file" value="$(arg config_file)" />

    <!--Parameters-->
    <param name='runlc' value='false' />

  </node>

</launch>
